### ISSUE TITLE
[REQUEST] Add support for the standard ext-foreign-toplevel-list

### ISSUE THREAD

**Issue by keithbowes** [lbonn/rofi#137]
### Before opening a feature request

- [X] I checked the *next* branch to see if the feature has already been implemented
- [X] I searched existing reports to see if it is already requested.

### What is the user problem or growth opportunity you want to see solved?

Rofi should support the standard ext-foreign-toplevel-list (introduced in wayland-protocols 1.32) in addition to wlr-foreign-toplevel-management.

### How do you know that this problem exists today? Why is this important?

[lswt](https://git.sr.ht/~leon_plickat/lswt) supports both, preferring ext-foreign-toplevel-list. Rofi could do the same. I've considered implementing it myself and sending a PR but I'd like to know it would be accepted before going to the trouble.

### Who will benefit from it?

Anyone using a Wayland compositor that doesn't support the proprietary wlr-foreign-toplevel-management protocol.

### Rofi version (rofi -v)

Version: 1.7.5+wayland3

### Configuration

https://gist.github.com/keithbowes/311ea3b89cabc843795954dffdfbf7e9#file-rofi-gist

### Additional information

I don't know why I need to go to the trouble of creating a Gist when this doesn't depend on configuration, but OK.

**Comment by lbonn** [comment]
@keithbowes yes, it would be worthwhile to go in the direction of using mainlined wayland protocols
